forge-gui/res/cardsfolder/upcoming/slick_sequence

```diff
@@ -0,0 +1,7 @@
+Name:Slick Sequence
+ManaCost:U R
+Types:Instant
+A:SP$ DealDamage | ValidTgts$ Any | NumDmg$ 2 | SubAbility$ TrigDraw | SpellDescription$ CARDNAME deals 2 damage to any target. If you’ve cast another spell this turn, draw a card.
+SVar:TrigDraw:DB$ Draw | ConditionCheckSVar$ X | ConditionSVarCompare$ GE2
```

Comment:
I think with the condition you should just add `StackDescription$ SpellDescription | SpellDescription$ If you’ve cast another spell this turn, draw a card.`

And then trim the SpellDesc in line 4